Gems/Atom/Feature/Common/Code/Source/FrameCaptureSystemComponent.cpp

```diff
@@ -97,7 +97,12 @@ namespace AZ
             Utils::PngFile image = Utils::PngFile::Create(readbackResult.m_imageDescriptor.m_size, format, *buffer);
 
             Utils::PngFile::SaveSettings saveSettings;
-            saveSettings.m_compressionLevel = r_pngCompressionLevel;
+
+            if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
+            {
+                console->GetCvarValue("r_pngCompressionLevel", saveSettings.m_compressionLevel);
+            }
+
```

Comment:
This seems likely to be a common piece of code, can we make a function that wraps this so it's just
```c++  
ReadCVarValue("r_pngCompressionLevel", saveSettings.m_compressionLevel);

template<typename T> void ReadCVarValue(const char* cvarNameStr, T& value)  
{  
    if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)  
    {  
        console->GetCvarValue(cvarNameStr, value);  
     }  
}  
```
Bonus points for a 2nd function that reads multiple values.